### PR TITLE
Global Notification support - pt2

### LINKED ISSUE #794: Global Notification Support
Not sure if I'm using the right language for this, but I wanted to work on having the same notification support on the app as on the website. The inbox notifications do not currently appear to be working, and having guild and party notifications would be nice as well. I would do the code for this if it were something we were looking to implement.

### PR DESCRIPTION
**Pull request by cvuorinen**
Continuation of #1116 

Fixes #794 

my Habitica User-ID: fa756acf-9127-4a3c-8dac-8ff627d30073

### CODE REVIEW

**Comment by cvuorinen** [comment]
I'm still waiting for an answer on one question in Zeplin (about boss quest display, should it be consistent with other places and display boss health, or display "Defeat: boss name" as in the design, but then there are no translations for the word "defeat" as it is not used anywhere else). I'm also still working on adding translations for all the new strings (that exist in website translations).

This is what it looks like ATM:

Screenshot_1559712883

**Comment by cvuorinen** [comment]
I've added all the translations found in the website translations now. And if the boss name in quest notification is ok, then this is ready for review.

File: Habitica/res/values-de/strings.xml [hunk 1]
```diff
@@ -790,15 +790,27 @@ Die Quest-Schriftrolle wird an den Quest-Besitzer zurückgegeben.</string>
...
+  <string name="username_level">\@%s ・Lvl %d</string>
+  <string name="verification_pet">Eines dieser Veteranen-Haustiere wartet auf Dich wenn Du die Bestätigung abgeschlossen hast!</string>
+  <string name="gift_subscription">Abonnement verschenken</string>
+  <string name="create_party">Gruppe erstellen</string>
+  <string name="notifications">Mitteilungen</string>
+  <string name="no_notifications_title">Du bist auf dem Laufenden!</string>
+  <string name="no_notifications_text">Die Mitteilungsfeen geben dir eine heftige Runde Applaus! Gut gemacht!</string>
+  <string name="dismiss_all">Alle entfernen</string>
+  <string name="new_bailey_update">Neues Update von Bailey!</string>
+  <string name="new_msg_guild"><![CDATA[<b>%1$s</b> hat neue Beiträge]]></string>
```
Comment (phillipthelen) [inline Habitica/res/values-de/strings.xml]: Have you tested this? I'm not sure if the `<![CDATA[` part works?

Comment (phillipthelen) [reply Habitica/res/values-de/strings.xml]: Nevermind. Just noticed that this is also in the english versions and also in the ones already merged in and it works.

**Pull request merged**